.github/workflows/ci.yml

```diff
@@ -389,6 +389,45 @@ jobs:
                 "./build_tools/scripts/check_cuda.sh
                 ./build_tools/scripts/check_vulkan.sh
                 ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}"
+                
```

Comment:
remove trailing whitespace here (you can configure your editor to do this for you)

Reply:
(see the failing lint check: https://github.com/iree-org/iree/actions/runs/8976316866/job/24652921251?pr=17293)